docs/_data/prague-2021-config.yaml

```diff
@@ -0,0 +1,172 @@
+name: Prague
+
+shortcode: prague
+year: 2021
+city: Prague
+local_area: Czech Republic
+area: Europe
+area_adj: European
+tz: CEST
+email: prague@writethedocs.org
+color: gold
+rgb: 253,185,19
+hex: fdb913
+
+photos:
+  default: _static/conf/images/headers/prague-group.png
+
+buttons:
+  top:
+    - text: Welcome to our conference
+      link: /news/welcome
+#    - text: Date and ticket information
+#      link: /news/announcing-virtual-conf-dates-tickets
+    # - text: Read our conference guide
+    #   link: /welcome-wagon
+    # - text: See the Schedule!
+    #   link: /schedule
+#    - text: Buy a Ticket!
+#      link: /tickets
+    - text: Sponsor us
+      link: /sponsors/prospectus
+    #- text: Submit a Talk
+    #  link: /cfp
+    #- text: See the talks!
+    #   link: /speakers
+    # - text: See the Schedule!
+    #   link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+  bottom:
+     - text: Welcome to our conference
+       link: /news/welcome
+    # - text: See the Schedule!
+    #   link: /schedule
+#     - text: Date and ticket information
+#       link: /news/announcing-virtual-conf-dates-tickets
+     - text: Sponsor the conference
+       link: /sponsors/prospectus
+    # - text: See the talks!
+    #   link: /speakers
+#     - text: Buy a Ticket!
+#       link: /tickets
+    # - text: Prepare with the Welcome Wagon
+    #   link: /welcome-wagon
+    # - text: See the schedule!
+    #   link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
+  corporate:
+    price: €125
+  independent:
+    price: €75
+  student:
+    price: €25
+
+sponsorship:
+  second_draft:
+    price: €1,500
+  publisher:
+    price: €3,500
+  patron:
+    price: €7,500
+  keystone:
+    price: €13,500
+
+
+date: # how do we handle these? human readable would be nice.
+  main: "**April 25-27, 2021, online**"
+  short: Oct 3-5, 2021
+  tickets_live: "XXXX 2021"
+  month: October
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Main Conference
+    date: October 4-5
+    summary: The main days of the conference. We will be running one main track, and an <a href="/conf/prague/2021/unconference/">Unconference</a>.
+    icon: conference
+  # These are listed in the schedule
+  day_one:
+    event: Hike
```

Comment:
Where is the boat ride? Should we even have it since it's DIY?

Reply:
This is what we had last year apparently:
https://github.com/writethedocs/www/blob/master/docs/_data/prague-2020-config.yaml#L94-L100